lib/view_component/rails/tasks/view_component.rake

```diff
@@ -6,6 +6,6 @@ namespace :view_component do
   task statsetup: :environment do
     require "rails/code_statistics"
 
-    ::STATS_DIRECTORIES << ["ViewComponents", ViewComponent::Base.view_component_path]
+    ::STATS_DIRECTORIES << ["ViewComponents", ViewComponent::Base.generate.component_path]
```

Comment:
I'm not sure this change is needed. The `view_component_path` setting still exists.

Reply:
That must've crept through - I thought I'd removed this when I dropped ceed17032b57346ce8eff5bf302c834b403ee1f1. I'll look into it.